docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,166 @@
++------------------------+---------------------------------------------------------------------------------------------+
+| Light Horizontal Angle | Sets the rotation around the zone's y axis and starts in its -z direction.                  |
++------------------------+---------------------------------------------------------------------------------------------+
+| Light Vertical Angle   | Sets the rotation around the zone's x axis and starts in its -z direction.                  |
++------------------------+---------------------------------------------------------------------------------------------+
+| Cast Shadows           | This enables shadows, while still respecting the "Cast Shadows" property of other entities. |
++------------------------+---------------------------------------------------------------------------------------------+
+| Shadow Bias            | This controls shadow bias and needs to be fine tuned. Too low values can cause so called    |
+|                        | "shadow acne", while too high values can cause so called "peter panning".                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Shadow Max Distance    | Sets the maximum distance in meters from your view at which shadows will be shown.          |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^^^
+Skybox
+^^^^^^
+
+Skyboxes are used to define the background of a zone. There are two types of skybox images that can be used.
+The aspect ratio is used to define the type of image. Images with a differing aspect ratio are not displayed.
+
+Types:
+
+* Cubemaps have an aspect ratio of 4:3 and consist of 6 sides, which are projected onto a cube.
+
+.. image:: _images/cubemap-skybox.png
+
+* Equirectanglar images have an aspect ratio of 2:1 and are projected onto a sphere.
+
+.. image:: _images/equirectanglar-skybox.jpg
+
+^^^^^^^^^^^^^
+Ambient Light
+^^^^^^^^^^^^^
+
+Ambient lighting illuminates the view according to the brightness and color of a texture.
+Accepted properties are:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Ambient Intensity      | This sets the intensity of the illumination.                                                |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ambient Source         | This takes the URL of a texture similar to the skybox. So similar to the skybox, that you   |
```

Comment:
It should be "URL of" rather than "URL to" since it is "the Uniform Resource Locator of ...".
Not only the texture layout is the same, but also the maximum resolution.

Reply:
Change to:

> This is a URL of a texture with the same qualities of a skybox. You can also use the actual skybox URL in this field. One scenario where this could be useful would be a space station that is illuminated my multiple suns and reflections from different colored planets.